### PR TITLE
Ignore all unknown JSON fields during de-seralization

### PR DESCRIPTION
**Pull request by jnazander**
Fix for #149

### CODE REVIEW

File: protostuff-json/src/test/java/io/protostuff/UnknownFieldTest.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+import io.protostuff.runtime.RuntimeSchema;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Test for ignoring unknown fields during de-seralization
+ */
+public class UnknownFieldTest {
```
Comment (kshchepanovskyi) [inline protostuff-json/src/test/java/io/protostuff/UnknownFieldTest.java]: Please rename it to `JsonUnknownFieldTest`.

File: protostuff-json/src/main/java/io/protostuff/JsonInput.java [hunk 2]
```diff
@@ -197,12 +196,12 @@ public JsonInput reset()
                         // skip the scalar elements
                         while (parser.nextToken() != END_ARRAY)
                             ;
-
-                        continue;
                     }
-
-                    throw new JsonInputException("Unknown field: " + name + " on message " +
-                            schema.messageFullName());
+					else
```
Comment (kshchepanovskyi) [inline protostuff-json/src/main/java/io/protostuff/JsonInput.java]: Please reformat code - change tabs to spaces. Formatter settings: https://github.com/protostuff/protostuff/blob/master/styleGuidelines.xml (eclipse)

Comment (jnazander) [inline protostuff-json/src/main/java/io/protostuff/JsonInput.java]: Done

File: protostuff-json/src/test/java/io/protostuff/UnknownFieldTest.java (hunk 1 above)
Comment (jnazander) [inline protostuff-json/src/test/java/io/protostuff/UnknownFieldTest.java]: Renamed

**Comment by kshchepanovskyi** [comment]
Acceptable :)

**Pull request merged**